packages/signature-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0]
```

Comment:
Some of the changes here were included in the changes for `message-manager`. Should we list them here?

``` suggestion
## [3.0.0]
## Added
- Add support for deferred signing ([#1364](https://github.com/MetaMask/core/pull/1364))
  - If the parameter `deferSetAsSigned` is set, the message won't be set as signed when the keyring is asked to sign it
- Add `setPersonalMessageInProgress` and `setTypedMessageInProgress` methods ([#1339](https://github.com/MetaMask/core/pull/1339))
```